### ISSUE TITLE
Getting the same output multiple time when running `accent sync`

### ISSUE THREAD

**Issue by louim** [mirego/accent-cli#10]
Hey! Do you know what could be causing this kind of output duplication? Is seems to match the number of language I have set for this project in Accent.

![capture d ecran 2018-07-04 a 12 02 07](https://user-images.githubusercontent.com/923718/42286906-58617044-7f82-11e8-8f2e-d2509bbad688.png)

**Comment by simonprev** [comment]
Yup that’s exactly it 😄 It list all operations that will be applied for every languages. But `new` and `remove` will effectively always be the same for every languages.

This should be handled by the cli and not the API
